root/static/styles/extra/homepage.less

```diff
@@ -36,6 +36,91 @@
     }
 }
 
+#about {
+    .about {
+        background: #f3f3f3;
+        padding: 30px 30px 0 30px;
+
+        .heading {
+            font-size: 2rem;
+            font-weight: 600;
+        }
+
+        .color-purple {
+            color: #ba478f;
+        }
+
+        .color-orange {
+            color: #eb743b;
+        }
```

Comment:
Makes sense. I would like to add that the CSS work here isn't final yet because we need to define the color palette for MB and use variables for colors such that we can switch the theme for light/dark. Most probably once the entire homepage is done, I'll add a PR that defines the theme and makes the page relevant for themes. Doing this in this PR will be unnecessarily confusing so reserving it for later :)
Also, these suggestions have been made!